canonical/ubuntu-com-security-api
# Cache notice API endpoints for 6 hours

## Done

Add some cache to the notice the endpoints to alleviate the load on the pods.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8030/
- Go to the endpoints and check max-age is set to 6 hours.
